http/users/http.go

```diff
@@ -0,0 +1,127 @@
+package users
+
+import (
+	"user-curd/models"
+	"user-curd/services"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+type Handler struct {
+	Sev services.Service
+}
+
+func (serv Handler) Search(w http.ResponseWriter, r *http.Request) {
```

Comment:
Usually we name our method receivers with the first letter of the receiver, in this case h